app/components/UI/Stake/components/StakingEarnings/StakingEarningsHistory/StakingEarningsTimePeriod/StakingEarningsTimePeriod.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from 'react';
+import { View } from 'react-native';
+import Button, {
+  ButtonVariants,
+} from '../../../../../../../component-library/components/Buttons/Button';
+import Text, {
+  TextVariant,
+} from '../../../../../../../component-library/components/Texts/Text';
+import { useStyles } from '../../../../../../../component-library/hooks';
+import styleSheet from './StakingEarningsTimePeriod.styles';
+import SkeletonPlaceholder from 'react-native-skeleton-placeholder';
+
+export enum DateRange {
+  DAILY = '7D',
+  MONTHLY = 'M',
+  YEARLY = 'Y ',
```

Comment:
Probably a typo! The additional space moves the `Y` off-center
```suggestion
  YEARLY = 'Y',
```